ci/docker/alpine-linux-3.18-cpp.dockerfile

```
@@ -39,8 +39,8 @@ RUN apk add \
         grpc-dev \
         gtest-dev \
         libxml2-dev \
-        llvm13-dev \
-        llvm13-static \
+        llvm15-dev \
+        llvm15-static \
```

Let's try 16 because the default `clang` version is 16:

```suggestion
        llvm16-dev \
        llvm16-static \
```